tasties_app/tests/conftest.py

```diff
@@ -183,3 +183,41 @@ def recipe():
 def signed_up_credentials():
     user = User.objects.create_user(username=VALID_USER, email=VALID_EMAIL, password=VALID_PASSWORD)
     user.save()
+
+
+@pytest.fixture
+def recipes_filtered_by_salad():
+
+    """
+    This fixture create the data we need for tests
+    and save it in the data base.
+    """
+    user1 = User.objects.create_user(username="john", password="password")
+    test_data = [
```

Comment:
Why should you bother with creating this list, if you're only using it to construct a `Recipe` locally in this fixture? Why not just assign everything directly in the constructor for `recipe1`?